### PR TITLE
refactor : clean dmi file extract spritten hiden inside other sprites direction. removing sprite change from dmm files.

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
clean dmi file extract sprite hidden inside other sprites direction. removing sprite change from dmm files.

# Explain why it's good for the game
it's good for mappers and our common sanity.
sprite shouldn't be hidden into sprite and sprite shouldn't be change in DMM files....
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
refactor: clean dmi file extract sprite hidden inside other sprites direction. removing sprite change from dmm files.
/:cl:

### CODE REVIEW

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 1]
```diff
@@ -332,9 +328,9 @@
 /area/fiorina/station/medbay)
 "ajh" = (
 /obj/item/device/flashlight/on,
-/turf/open/floor/prison{
+/turf/open/floor/prison/darkbrown2angle{
 	dir = 1;
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:332]:
```suggestion
	dir = 1
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 2]
```diff
@@ -332,9 +328,9 @@
...
-	icon_state = "darkbrowncorners2"
+
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:333]:
```suggestion
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 3]
```diff
@@ -1109,17 +1093,17 @@
...
 /area/fiorina/station/medbay)
 "aDl" = (
 /obj/structure/machinery/vending/cola,
 /obj/structure/prop/souto_land/streamer{
 	dir = 1;
 	pixel_y = 24
 	},
-/turf/open/floor/prison{
+/turf/open/floor/prison/darkbrown2{
 	dir = 8;
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1105]:
```suggestion
	dir = 8
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 4]
```diff
@@ -1109,17 +1093,17 @@
...
-	icon_state = "darkbrown2"
+
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1106]:
```suggestion
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 5]
```diff
@@ -1487,15 +1469,15 @@
 /obj/structure/platform_decoration{
 	dir = 4
 	},
-/turf/open/floor/prison{
-	dir = 5;
-	icon_state = "darkbrown2"
+/turf/open/floor/prison/darkbrown2corner{
+	dir = 4;
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1473]:
```suggestion
	dir = 4
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 6]
```diff
@@ -1487,15 +1469,15 @@
...
+
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1474]:
```suggestion
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 7]
```diff
@@ -1487,15 +1469,15 @@
...
+	dir = 4;
+	
 	},
 /area/fiorina/station/park)
 "aNn" = (
-/turf/open/floor/prison{
-	dir = 10;
-	icon_state = "darkbrown2"
+/turf/open/floor/prison/darkbrown2corner{
+	dir = 1;
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1479]:
```suggestion
	dir = 1
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 8]
```diff
@@ -1487,15 +1469,15 @@
...
+
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1480]:
```suggestion
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 9]
```diff
@@ -1612,17 +1592,17 @@
 /area/fiorina/station/flight_deck)
 "aQM" = (
 /obj/effect/landmark/structure_spawner/xvx_hive/xeno_door,
-/turf/open/floor/prison{
+/turf/open/floor/prison/darkbrown2{
 	dir = 4;
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1596]:
```suggestion
	dir = 4
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 10]
```diff
@@ -1612,17 +1592,17 @@
...
-	icon_state = "darkbrown2"
+
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1597]:
```suggestion
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 11]
```diff
@@ -2778,19 +2750,19 @@
 	},
 /area/fiorina/station/medbay)
 "bDN" = (
-/turf/open/floor/prison{
+/turf/open/floor/prison/darkbrown2{
 	dir = 4;
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:2754]:
```suggestion
	dir = 4
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 12]
```diff
@@ -2778,19 +2750,19 @@
...
-	icon_state = "darkbrown2"
+
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:2755]:
```suggestion
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 13]
```diff
@@ -2778,19 +2750,19 @@
...
 	},
 /area/fiorina/station/park)
 "bDU" = (
 /obj/item/stack/rods,
 /turf/open/floor/plating/prison,
 /area/fiorina/tumor/aux_engi)
 "bEn" = (
-/turf/open/floor/prison{
+/turf/open/floor/prison/darkbrown2{
 	dir = 4;
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:2764]:
```suggestion
	dir = 4
```

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm [hunk 14]
```diff
@@ -2778,19 +2750,19 @@
...
-	icon_state = "darkbrown2"
+
```
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:2765]:
```suggestion
```

**Review by Drulikar — CHANGES_REQUESTED** [review]
See my suggestions and repeat. It's not that it was missing new lines, it's that when you use ; that indicates there is supposed to be another var edit on the following line.

**Comment by Huffie56** [comment]
i saw in mapbot that i made an oupsy
will fix it later...

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm
Comment (Nanu308) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1]: bilde
You're making visual changes to the tiles used. Make sure you keep it the same if this is just a refractor.

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm
Comment (Huffie56) [reply maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1]: fixed no map change as far as i can tell now.

**Comment by Huffie56** [comment]
i remove all the icon-state for floors for fiorina their is like 600 random icon state to handle for the map....

**Comment by Drulikar** [comment]
Sprites and mapping aren't generally things I can approve/merge but there is nothing obviously wrong with the changes at the moment.

**Comment by Huffie56** [comment]
> 

thanks anyway for checking it...

**Review by harryob — APPROVED** [review]
bless you

**Comment by Huffie56** [comment]
> bless you

this is a pain to do won't do this every month....

**Pull request merged**